Source/Core/InputCommon/ControlReference/ExpressionParser.cpp

```diff
@@ -488,6 +510,32 @@ class UnarySinExpression : public UnaryExpression
   std::string GetFuncName() const override { return "Sin"; }
 };
 
+class UnaryWhileExpression : public UnaryExpression
+{
+public:
+  UnaryWhileExpression(std::unique_ptr<Expression>&& inner_) : UnaryExpression(std::move(inner_)) {}
+
+  ControlState GetValue() const override
+  {
+    constexpr int MAX_REPS = 10000;
```

Comment:
Commit message says 1000, constant says 10000. Which one is it?

Reply:
;)